model/index.js

```diff
@@ -64,13 +60,36 @@ module.exports = yeoman.Base.extend({
 
   addNullDataSourceItem: actions.addNullDataSourceItem,
 
+  setBluemixDatasourceState: function() {
+    if (this.options.bluemix) {
+      var configPath = this.destinationPath('.bluemix/datasources-config.json');
+      if (!fs.existsSync(configPath)) {
+        this.log('datasources-config.json not found');
+        process.exit();
```

Comment:
It's not ideal, but I cannot find a graceful way in Yeoman to report the error and terminate the generator. Unfortunately `done(err)` prints ugly stack trace. I'm not sure if `process.exit()` will mess up the terminal.